### ISSUE TITLE
[Validator] [Twig] added magic method __isset to File Constraint class

### ISSUE THREAD

**Issue by loru88** [symfony/symfony#24512]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | yes
| BC Break report? | no
| RFC?             | yes
| Symfony version  | all

In my project I get assert constraints from one of my entity and I use this value in my front end via Twig.
I faced a problem with the property [$maxSize](https://github.com/symfony/symfony/blob/2ab092034b806dab7aa52de542ac15b88a4b7066/src/Symfony/Component/Validator/Constraints/File.php#L60) of the File Constraint.

As this property is protected I cannot access it via Twig because the magic method `__isset` is missing, as I read in [twig documentation](https://twig.symfony.com/doc/2.x/recipes.html#using-dynamic-object-properties).

so I propose to add this piece of code to be able to get this value accessible in Twig:

```php
    public function __isset($option)
    {
        if ('maxSize' === $option) {
            return true;
        }

        return parent::__isset($option);
    }
```

this solution work for sure as I tested it, but my question is why this is not a simple public property as the rest of the File Constraint class properties?

**Comment by loru88** [comment]
I did a pull request for this: #24519

**Comment by javiereguiluz** [comment]
Fixed by #24519.
